web/src/main/webapp/ui/dialog.ts

```diff
@@ -740,6 +740,39 @@ export class NotifyDialog extends DialogBase {
     }
 }
 
+
+/**
+ * Bookmark notification with a copy link button.
+ */
+export class NotifyBookmarkDialog extends NotifyDialog {
+    constructor(title: string, url: string | null, toolTip: string) {
+        super(title, null, toolTip);
+
+        const nodrag = d3drag()
+            .on("start", () => this.dragEnd());
+
+        const urlBox = makeInputBox(url) as HTMLInputElement;
+        urlBox.style.width = "-webkit-fill-available";
```

Comment:
webkit? In general we try to write portable code across all browsers. Is there a portable version of this attribute?